### DISCUSSION TITLE
Port 2 LO Attenuator

### DISCUSSION THREAD

**Discussion by nateof8** [jankae/LibreVNA#246] [Q&A] [answered]
Jan, why is there a 10dB pi attenuator on the LO for the Port 1 and Reference mixers but not the Port 2 mixer?

**Comment by jankae** [comment, marked as answer]
That is an attempt to improve port isolation. Some signal was/is leaking through the LO lines between the port 1 and reference mixers (because the LO lines are close to each other). The attenuator is supposed to reduce the leakage through that path. Although not very successful, it did make a small difference so I kept the attenuators.

Port 2 is not as critical because the physical distance to the reference receiver is larger.

**Reply by nateof8** [reply]
Thanks for the reply.  For your awareness, I did notice that there is a small break in the main reference plane (looks like TP15) for the Reference 1.LO between the MAX2871 and the ADL5801.  Does this LO run up to the max frequency of 6GHz?
